src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -36,9 +32,18 @@
             (update-in [:wallet :ui :send] dissoc :route)
             (update-in [:wallet :ui :send] dissoc :loading-suggested-routes?))}))
 
+(rf/reg-event-fx :wallet/select-send-account-address
+ (fn [{:keys [db]} [{:keys [address stack-id]}]]
+   {:db (-> db
+            (assoc-in [:wallet :ui :send :send-account-address] address)
+            (update-in [:wallet :ui :send] dissoc :to-address))
+    :fx [[:navigate-to-within-stack [:wallet-select-asset stack-id]]]}))
+
 (rf/reg-event-fx :wallet/select-send-address
  (fn [{:keys [db]} [{:keys [address stack-id]}]]
-   {:db (assoc-in db [:wallet :ui :send :to-address] address)
+   {:db (-> db
+            (assoc-in [:wallet :ui :send :to-address] address)
+            (update-in [:wallet :ui :send] dissoc :send-account-address))
     :fx [[:navigate-to-within-stack [:wallet-select-asset stack-id]]]}))
```

Comment:
I think these events only differ in the key where they store the selected address. But why do we need them? I think we could mark in `:ui :send` a boolean instead :thinking: 

about the names of the events `:wallet/select-send-address` and `:wallet/select-send-account-address` are very similar, maybe we could set a different name more descriptive to understand them better.